server/pkg/auth/auth.go

```diff
@@ -83,12 +83,17 @@ func CreateSessionToken() string {
 }
 
 // SetCookie creates an http cookie and sets it in the response
-func SetCookie(w http.ResponseWriter, name, token string) {
-	// TODO: Implement remember me functionality (MaxAge: 0)
+func SetCookie(w http.ResponseWriter, name, token string, rememberMe bool) {
+	maxAge := SessionMaxAge
+
+	if rememberMe == true {
+		maxAge = 31536000000 // 1000 years in seconds
```

Comment:
Can you make this a constant and reference here instead of just putting the number here? Same as the way `SessionMaxAge` is set.

Reply:
Done!